records_mover/records/pandas/prep_for_csv.py

```diff
@@ -21,15 +22,13 @@ def _convert_series_or_index(series_or_index: T,
                              records_format: DelimitedRecordsFormat,
                              processing_instructions: ProcessingInstructions) -> Optional[T]:
     if field.field_type == 'date':
-        if not isinstance(series_or_index[0], pd.Timestamp):
-            logger.warning(f"Found {series_or_index.name} as unexpected type "
-                           f"{type(series_or_index[0])}")
-        else:
+        # TODO: Simplify code below
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker